src/components/cards/TradeCard/TradeRoute.vue

```diff
@@ -427,13 +427,17 @@ export default defineComponent({
     function getPoolLink(id: string): string {
       const { chainId } = getConfig();
       const prefixMap = {
-        1: '',
+        1: 'app.',
         42: 'kovan.'
       };
       const prefix = prefixMap[chainId] || '';
-      return props.sorReturn.isV1swap
-        ? `https://${prefix}pools.balancer.exchange/#/pool/${id}`
-        : `https://${prefix}app.balancer.fi/#pool/${id}`;
+      if (props.sorReturn.isV1swap && chainId === 1) {
```

Comment:
We can now do something like `chainId === Network.MAINNET` (imported from `network.ts`).